pom.xml

```diff
@@ -623,6 +625,66 @@
             </upToDateChecking>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.openrewrite.maven</groupId>
+          <artifactId>rewrite-maven-plugin</artifactId>
+          <version>6.24.0</version>
+          <configuration>
+            <exportDatatables>true</exportDatatables>
+            <skipMavenParsing>true</skipMavenParsing>
+            <activeRecipes>
+              <recipe>org.checkstyle.openrewrite.SanityCheck</recipe>
+            </activeRecipes>
+            <exclusions>
+              <exclusion>**.ci-temp**</exclusion>
+              <exclusion>**.template</exclusion>
```

Comment:
why added ?

Reply:
because of new recipe, extract.

Reply:
Let's create new PR with new recipes and excludes for them, it will be very clear by commit content why it is added .
With smaller commit, it is more clear why lines are added and we might not need comment, and commit message explained